source/extensions/omni.isaac.lab/omni/isaac/lab/managers/observation_manager.py

```diff
@@ -83,13 +122,22 @@ def active_terms(self) -> dict[str, list[str]]:
         return self._group_obs_term_names
 
     @property
-    def group_obs_dim(self) -> dict[str, tuple[int, ...]]:
-        """Shape of observation tensor in each group."""
+    def group_obs_dim(self) -> dict[str, tuple[int, ...] | list[tuple[int, ...]]]:
+        """Shape of observation tensor in each group.
+
+        If the terms are concatenated, the value is a single tuple representing the shape of the
```

Comment:
Can you describe the dictionary return? I'm a little confused here I think it's only because you're not describing the keys of the dictionary-type return

Reply:
Fixed.